Cargo.toml

```diff
@@ -8,7 +8,9 @@ members = [
     "ddprof-ffi",
     "ddprof-profiles",
     "ddcommon",
+    "ddcommon-ffi",
```

Comment:
Actually, not that bad of an idea. I don't think it would cause issues.

We could probably have an ffi module in ddcommon, ddprof, etc.. that exposes only what needs to be exposed.